js/form.js

```diff
@@ -72,9 +80,53 @@ timeOut.addEventListener('change', () => {
   timeIn.value = timeOut.value;
 });
 
-adForm.addEventListener('submit', (evt) => {
-  if (!pristine.validate()) {
-    evt.preventDefault();
-  }
+const resetForm = () => {
+  priceRoom.placeholder = 1000;
+  address.placeholder = '35.68622, 139.77074';
```

Comment:
тут значение нужно устанавливать, не плейсхолдер

Reply:
Можешь переместить в очередь после основного потока, например
```suggestion
  setTimeout(() => {
    address.value = '35.68622, 139.77074';
  }, 0)
```